Source/WebCore/rendering/style/RenderStyle.cpp

```diff
@@ -525,8 +525,16 @@ bool RenderStyle::descendantAffectingNonInheritedPropertiesEqual(const RenderSty
         || m_nonInheritedData->miscData.ptr() == other.m_nonInheritedData->miscData.ptr())
         return true;
 
-    return m_nonInheritedData->miscData->alignItems == other.m_nonInheritedData->miscData->alignItems
-        && m_nonInheritedData->miscData->justifyItems == other.m_nonInheritedData->miscData->justifyItems;
+    if (m_nonInheritedData->miscData->alignItems != other.m_nonInheritedData->miscData->alignItems)
```

Comment:
Maybe a good idea to have a local variable for miscData given how many reads we have

Reply:
hmm we generally don't use local variables for `DataRef` in this file – so will leave as-is unless @anttijk also thinks we should